jbpm-designer-api/org.jbpm/jbpm-designer-api/.zanata-cache/etag-cache.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<etag-cache/>
```

Comment:
@mbiarnes This file probably doesn't need to be part of this commit and/or should be in .gitignore?